app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -201,12 +199,14 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
   return (
     <>
       <PageHeading>{t('in_person_proofing.headings.location')}</PageHeading>
-      {arcgisSearchEnabled && (
+      <div aria-live="polite">
         <AddressSearch onAddressFound={handleFoundAddress} registerField={registerField} />
-      )}
-      <p>{t('in_person_proofing.body.location.location_step_about')}</p>
-      {locationsContent}
-      <BackButton onClick={toPreviousStep} />
+      </div>
+      <div role="status">
+        <p>{t('in_person_proofing.body.location.location_step_about')}</p>
+        {locationsContent}
```

Comment:
the text juts up against the search button causing the first letter to be hard to read. We should check in w/ design to see if they have a rec on spacing.

Reply:
also, if the line is too long the zip code wraps but in a way where it has a lot of spacing above the 2nd line 
<img width="530" alt="Screen Shot 2022-12-09 at 12 44 58 PM" src="https://user-images.githubusercontent.com/20867088/206761753-b5228e13-3828-44d3-b744-fc00684f54ae.png">

Reply:
Matt commented in slack that this is unstyled as it's just for debugging. so please ignore my comments